src/core/best-practices.js

```diff
@@ -49,12 +50,18 @@ export function run() {
 
     // Make the advisement box
     container.classList.add("advisement");
-    const title = hyperHTML`${localizedBpName.cloneNode(true)}: ${bp}`;
+    const title = html`
+      ${localizedBpName.cloneNode(true)}: ${bp}
+    `;
     container.prepend(...title.childNodes);
   });
   if (bps.length) {
     if (bpSummary) {
-      bpSummary.appendChild(hyperHTML`<h2>Best Practices Summary</h2>`);
+      bpSummary.appendChild(
+        html`
+          <h2>Best Practices Summary</h2>
+        `
+      );
```

Comment:
I'm not quite sure I like this though, because there are certainly some situations where a single line becomes 5 lines. I wonder whether we have any option to make it any shorter. (Not requiring multiline for short-enough string would be great.)

Reply:
Is prettier forcing this to be multi-line?

Reply:
Yes. 😢

Reply:
https://github.com/prettier/prettier/issues/5588

Reply:
Filed https://github.com/prettier/prettier/issues/7101, because disabling the whole HTML formatting sound bad for me 👀